e2e/README.md

```
@@ -0,0 +1,36 @@
+# Interchaintest
+
+## Current ICTests
+| CLI | Description   |  |   |   |
```

Markdownlint (0.37.0)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)